src/main/java/com/epam/ta/reportportal/job/FlushingDataJob.java

```diff
@@ -171,14 +171,9 @@ private void deleteUser(User user) {
   }
 
   private void deleteProject(Project project) {
-    Set<Long> defaultIssueTypeIds = issueTypeRepository.getDefaultIssueTypes()
```

Comment:
Caused error: javax.persistence.EntityExistsException: A different object with the same identifier value was already associated with the session : [com.epam.ta.reportportal.entity.project.ProjectIssueType#com.epam.ta.reportportal.entity.project.ProjectIssueTypeKey@e7d3d6fa]